### ISSUE TITLE
Header

### ISSUE THREAD

**Issue by alicaudle** [alicaudle/ecommerce-pattern-library#6]
https://github.com/alicaudle/ecommerce-pattern-library/blob/master/_patterns/header/header.html#L12

@thomasjbradley

It won't let me click the products page.

**Comment by thomasjbradley** [comment]
*Re: products—*

I don’t see the `_products` folder or the stuff inside `_config.yml` — is it just not pushed?

**Comment by alicaudle** [comment]
I created a products.html file and placed it in the layouts folder. I guess it's supposed to have it's own products folder?
@thomasjbradley

**Comment by alicaudle** [comment]
I've placed it in its own folder and the products page still isn't linking.
@thomasjbradley

**Comment by thomasjbradley** [comment]
Oh, I see now! The `products.html` should be in the root folder, with `index.html`

**Comment by thomasjbradley** [comment]
Delete the `products` pattern folder.

**Comment by alicaudle** [comment]
It works now! It has been completed properly! The only thing I'm unsure of is how and where to place it in the _config.yml file. It's under "patternbot" but is it its own thing? so like "colors" and "rationales" I just create a "products" section with a title and description? cuz I don't have one for the index file, unless everything directly under patternbot is for the index file?
@thomasjbradley

**Comment by alicaudle** [comment]
https://github.com/alicaudle/ecommerce-pattern-library/blob/master/_config.yml

**Comment by thomasjbradley** [comment]
I don’t think you need the config file stuff yet. I thought you were onto collections—there’s a lesson on it when you get there.
